src/software/amazon/ion/Timestamp.java

```diff
@@ -1232,6 +1094,13 @@ public static Timestamp forSecond(int year, int month, int day,
     /**
      * Returns a Timestamp that represents the point in time that is
      * {@code millis} milliseconds from the epoch, with a given local offset.
+     * A default {@link GregorianCalendar} will be used to perform any
+     * arithmetic operations on the resulting Timestamp. NOTE: this means
```

Comment:
suggestion: use html tags to highlight the `NOTE`, e.g.
```
... Timestamp.

<p>
<strong> NOTE: </strong> this means ....
</p>
```

Reply:
Done.